### PR TITLE
doc/dev/crimson: note about script/ceph-debug-docker.sh

### PR DESCRIPTION
**Pull request by tchaikov**
Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins render docs

**Comment by tchaikov** [comment]
jenkins render docs

**Comment by tchaikov** [comment]
https://docs.ceph.com/ceph-prs/36674/dev/crimson/

File: doc/dev/crimson.rst [hunk 1]
```diff
@@ -203,3 +205,20 @@ the input, so you can also paste the log messages like::
...
+debugging crashes of teuthology based tests
+-------------------------------------------
+
+Unlike classic OSD, crimson does not print a human-readable backtrace when it
+handles fatal signals like `SIGSEGV` or `SIGABRT`. And it is more complicated
+when it comes to a stripped binary. So before planting a signal handler for
+those signals in crimson, we could to use `script/ceph-debug-docker.sh` to parse
+the addresses in the backtrace::
+
+  # assumeing you are under the source tree of ceph
```
Comment (ronen-fr) [inline doc/dev/crimson.rst:218]: 'assuming'

Comment (tchaikov) [reply doc/dev/crimson.rst:218]: #36702

**Pull request merged**